readme.md

```diff
@@ -23,129 +23,139 @@ Instructor introductions.
+
+# What Is Asking for Help? (10 minutes / 10:00)
+
+Asking for help is what students find hardest in WDI. It is also what ultimately determines whether or not students will be successful in this class.
 - Without fail, the students who do not ask for help do the poorest in the class.
-- Most importantly, **knowing how to ask for help and where to look for answers** is one of the biggest skills you can gain from this class that will be applicable as you prepare to join the professional workforce.
+- **Knowing how to ask for help and where to look for answers** are two of the biggest skills you can gain from this class and will be incredibly valuable as you prepare to join the professional workforce.
+
+#### Turn & Talk
 
-**Turn & Talk**: Everyone should take 30 seconds to think of a time when they've asked someone for help. It doesn't have to be tech-related. Then take 30 seconds to turn and share with your tablemates about those times. What were the outcomes?
+Take 30 seconds to think of a time when you have asked someone for help. It doesn't have to be tech-related. Then take another 30 seconds to turn and share with your tablemates about those times. What were the outcomes?
 
-<!-- ALB: turn into a postit note exercise? -->
+-----
 
 We have all asked for help. There's no shame in it. Feeling afraid to ask for help will be the biggest detriment in this class.
-- What makes asking for help effective?
-  - Asking a clear problem. "I don't know any of this" isn't a clear problem. "I tried this thing and got this error" is a clear problem.
-  - We'll ask you for help when:
-      - The question you're asking isn't constructive
-      - It's clear you haven't made an effort to find the answer:
-        - First, try it yourself
-        - Then, Google it
-        - Then, ask your neighbor
-        - Finally, ask an instructor
-
-# Break (10 minutes, til 10:10)
-
-# Course content (10 minutes, til 10:20) (Andy)
-
-### How the Class is Built
-
-GA's global team (including many of the instructors here) has built a baseline
-curriculum, which includes topics to cover, and a suggested order. As a local
-campus, we have built on and modified this curriculum based on own experiences
-as instructors, and the unique features of the DC job market. Each lesson is
+
+How can we ask for help efficiently?
+- Ask a clear problem. "I don't know any of this" isn't a clear problem. "I tried this thing and got this error" is a clear problem.
+- Instructors will be unable to help you if...
+  - The question you're asking isn't constructive
+  - It's clear you haven't made an effort to find the answer. This means taking the following steps...
+    1. First, try it yourself
+    - Then, Google it
+    - Then, ask your neighbor
+    - Finally, ask an instructor
+
+> Check out [this lesson plan](https://github.com/ga-wdi-lessons/effective_questions) if you'd like to learn more about how to ask effective questions.
+
```

Comment:
that's cool. could frame it though as don't worry about the code, just more about how the questions are asked. like you don't need to know rails to get the message  on how to write good questions